Makefile

```diff
@@ -22,3 +22,6 @@ license:
 	go install github.com/AbsaOSS/golic@v0.1.2
 	golic inject -c="2021 ABSA Group Limited"
```

Comment:
It's safer to stick with Absa actually. Sorry if I missed it in another license propagating PRs